addons/project/static/tests/project_task_state_selection.test.js

```diff
@@ -1,10 +1,10 @@
-import { expect, test } from "@odoo/hoot";
-import { click } from "@odoo/hoot-dom";
+import { expect, test, describe } from "@odoo/hoot";
+import { click, queryAllTexts } from "@odoo/hoot-dom";
 import { animationFrame } from "@odoo/hoot-mock";
 import { mountView } from "@web/../tests/web_test_helpers";
+import { defineProjectModels, ProjectTask } from "./project_models";
```

Comment:
```suggestion
import { expect, test, describe } from "@odoo/hoot";
import { click, queryAllTexts } from "@odoo/hoot-dom";
import { animationFrame } from "@odoo/hoot-mock";

import { mountView } from "@web/../tests/web_test_helpers";

import { defineProjectModels, ProjectTask } from "./project_models";
```
I think grouping the imports make them more readable 🤓